addons/packages/pinniped/0.22.0/bundle/vendir.lock.yml

```diff
@@ -0,0 +1,27 @@
+apiVersion: vendir.k14s.io/v1alpha1
+directories:
+- contents:
+  - git:
+      commitTitle: Make linter happy by deleting unused gocyclo
+      sha: 4d6a2af89419546832adfa8640eef22473e41526
+      tags:
+      - v0.12.1
```

Comment:
I believe this will auto-update when we get the version bumped correctly elsewhere.

Reply:
Fixed and pushed.